### PR TITLE
Add continuous integration example and move use cases to the appendix

### PR DESCRIPTION
**Pull request by arndt-s**
* Moves use cases to the appendix
* Adds new continuous integration use case

Let me know what you think!

### CODE REVIEW

File: draft-ietf-oauth-identity-chaining.md [hunk 1]
```diff
@@ -293,6 +284,19 @@ Authorization Servers SHOULD follow the OAuth 2.0 Security Best Current Practice
 
 --- back
 
+# Use cases
+
+This sections outlines some use cases the authors have identified where identity chaining as described in this document should be used. This section is not complete and other use cases not mentioned here are also valid.
```
Comment (bc-pi) [inline draft-ietf-oauth-identity-chaining.md:289]: Suggest a minor change in wording here (that also anticipates the likely merge of #76):

```suggestion
This sections outlines some use cases where the identity and authorization chaining described in this document can be applied. This section is not complete and other use cases not mentioned here are also valid.
```

**Review by bc-pi — CHANGES_REQUESTED** [review]
Thanks @arndt-s, I've got a small editorial suggestion but otherwise think the use case is definitely a worthwhile addition.

**Review by bc-pi — APPROVED** [review]

**Comment by bc-pi** [comment]
rendered preview of this PR can be seen https://drafts.oauth.net/oauth-identity-chaining/arndt-s/add-ci-use-case/draft-ietf-oauth-identity-chaining.html

File: draft-ietf-oauth-identity-chaining.md [hunk 2]
```diff
@@ -293,6 +284,19 @@ Authorization Servers SHOULD follow the OAuth 2.0 Security Best Current Practice
...
 --- back
 
+# Use cases
+
+This sections outlines some use cases where the identity and authorization chaining described in this document can be applied. This section is not complete and other use cases not mentioned here are also valid.
+
+## Preserve User Context across Multi-cloud, Multi-Hybrid environments
+A user attempts to access a service that is implemented as a number of on-premise and cloud-based microservices. Both the on-premise and cloud-based services are segmented by multiple trust boundaries that span one or more on-premise or cloud service environments. Every microservice can apply an authorization policy that takes the context of the original user, as well as intermediary microservices into account, irrespective of where the microservices are running and even when a microservice in one trust domain calls another service in another trust domain.
+
+## Continuous integration accessing external resources
```
Comment (bc-pi) [inline draft-ietf-oauth-identity-chaining.md:294]: per Dr. Kelley W Burgin:

```suggestion
## Continuous Integration Accessing External Resources
```

File: draft-ietf-oauth-identity-chaining.md [hunk 3]
```diff
@@ -293,6 +284,19 @@ Authorization Servers SHOULD follow the OAuth 2.0 Security Best Current Practice
...
+
+## Preserve User Context across Multi-cloud, Multi-Hybrid environments
+A user attempts to access a service that is implemented as a number of on-premise and cloud-based microservices. Both the on-premise and cloud-based services are segmented by multiple trust boundaries that span one or more on-premise or cloud service environments. Every microservice can apply an authorization policy that takes the context of the original user, as well as intermediary microservices into account, irrespective of where the microservices are running and even when a microservice in one trust domain calls another service in another trust domain.
+
+## Continuous Integration Accessing External Resources
+A continuous integration system needs to access external resources, for example to upload an artifact or to run tests. These resources are protected by different authorization servers. The identity information of the build, for example metadata such as commit hashes or repository, should be preserved and carried across the domain boundary. This not just prevents maintaining credentials it also allows fined grained access control at the resource.
```
Comment (PieterKas) [inline draft-ietf-oauth-identity-chaining.md:295]: replace "fined" with "fine"

**Pull request merged**